resources/js/components/LanguageSwitcher.vue

```diff
@@ -1,40 +1,37 @@
 <template>
-  <Dropdown placement="bottom-end">
-    <DropdownTrigger
-        :show-arrow="false"
-        class="h-10 w-full hover:text-primary-500"
-    >
-
-      <Icon type="language"  />
-      <span class="ml-1 font-bold hidden md:inline">{{selectedDisplay}}</span>
-    </DropdownTrigger>
+  <Dropdown placement="bottom-end" class="relative">
+    <Button variant="action" icon="globe-alt" class="text-primary-500 md:w-24" >
+      <span class="hidden md:inline">{{ selectedDisplay }}</span>
```

Comment:
Remove this because some language names are too long, so we removed it in the last version.

Reply:
I this comment i mean this line 
`      <span class="hidden md:inline">{{ selectedDisplay }}</span>
`